src/UI/UpButton.css

```diff
@@ -2,8 +2,9 @@
   display: none
 }
 
-@media (min-width: 768px) {
+@media (min-width: 844px) {
```

Comment:
Let's keep the previous viewport value because we want to keep it for iPad users.

Reply:
Sure